### PR TITLE
Initiate CI jobs on separate threads

### PR DESCRIPTION
**Pull request by casperboone**
Builds on #253. Review only 2c1525d and newer.
___

This PR adds the actual execution of CI jobs after the validation check requested event is broadcast. It does this on separate threads.

### CODE REVIEW

File: modules/validation-pipeline/ci-job/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/cijob/CIJobInitiator.kt [hunk 1]
```diff
@@ -1,19 +1,28 @@
...
+    }
+
     /**
      * Listens to [ValidationRequestReceivedEvent] events.
      */
     @EventListener
-    fun handleValidateRequestReceivedEvent(event: ValidationRequestReceivedEvent) {
-        println("Received event $event")
-    }
+    fun handleValidateRequestReceivedEvent(event: ValidationRequestReceivedEvent) =
```
Comment (JSAbrahams) [inline modules/validation-pipeline/ci-job/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/cijob/CIJobInitiator.kt]: It's nice to see that this method now actually does someting. But I don't really understand it, isn't `get` a blocking method? 
How can you submit multiple jobs to the `executorService`?

Comment (gandreadis) [reply modules/validation-pipeline/ci-job/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/cijob/CIJobInitiator.kt]: By submitting multiple validation request received events

File: modules/validation-pipeline/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/events/CIJobCompletedEvent.kt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package org.cafejojo.schaapi.validationpipeline.events
+
+import org.cafejojo.schaapi.validationpipeline.TestResults
+
+/**
+ * Event that can be broadcast after receiving a CI job run is completed.
```
Comment (gandreadis) [inline modules/validation-pipeline/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/events/CIJobCompletedEvent.kt]: "after receiving a CI job run is completed" -> "after receiving a 'CI job run is completed' event"
(current sentence doesn't parse)

Comment (casperboone) [reply modules/validation-pipeline/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/events/CIJobCompletedEvent.kt]: Was just a bad copy paste, receiving shouldn't be in this sentence ;)

Comment (gandreadis) [reply modules/validation-pipeline/src/main/kotlin/org/cafejojo/schaapi/validationpipeline/events/CIJobCompletedEvent.kt]: Ah ok, makes sense :)

**Pull request merged**